Assets/_BForBoss/_Character/Scripts/DashLinesEffectBehaviour.cs

```diff
@@ -0,0 +1,57 @@
+//-----------------------------------------------------------------------
+// <copyright file="DashLinesEffectBehaviour.cs" company="PERIGON GAMES">
+//     Copyright (c) PERIGON GAMES. 2020 All Rights Reserved
+// </copyright>
+//-----------------------------------------------------------------------
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Perigon.Character
+{
+    public class DashLinesEffectBehaviour : MonoBehaviour
+    {
+
+        [SerializeField] GameObject _dashEffect;
+        
+        [SerializeField] [Range(0,1)] float _lerpInStrength = 0.1f;
+        [SerializeField] [Range(0,1)] float _lerpOutStrength = 0.1f;
+        [SerializeField] float _duration = 0.1f;
+        private float _currentTime;
+        
+        [SerializeField] [Range(0,1)] float _maxAlpha = 1f;
+        private Material _speedLineMaterial;
+        
+        private void Start()
+        {
+            _speedLineMaterial = _dashEffect.GetComponent<RawImage>().material;
```

Comment:
You don't actually need to setup
`[SerializeField] GameObject _dashEffect;` 
if the component is on the same game object as this script. 

You can just call `GetComponent<RawImage>()` and it'll find it.

Reply:
Good to know! Changed in most recent commit.